### PR TITLE
ZOB-36 Add "Empty page" for the boxes and meal section

### PR DESCRIPTION
**Pull request by fabiawrc**
[ZOB-36](https://etnetera.atlassian.net/browse/ZOB-36) - Add "Empty page" for the boxes and meal section

### CODE REVIEW

File: lib/ui/screens/boxes_screen.dart [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
 import 'package:zachranobed/common/helper_service.dart';
 import 'package:zachranobed/extensions/build_context_extensions.dart';
 import 'package:zachranobed/models/box_movement_list_info.dart';
+import 'package:zachranobed/services/box_movement_service.dart';
 import 'package:zachranobed/ui/widgets/box_movement_list.dart';
 import 'package:zachranobed/ui/widgets/button.dart';
 
+import '../widgets/empty_page.dart';
+import '../widgets/error_page.dart';
+import '../widgets/loading_page.dart';
```
Comment (oleksandrbalan-etn) [inline lib/ui/screens/boxes_screen.dart:14]: Nit: Pls change to absolute imports 😇 

Also in `lib/ui/screens/donations_screen.dart` and `lib/ui/widgets/empty_page.dart`

File: lib/ui/widgets/empty_page.dart [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+              height: 250,
+              fit: BoxFit.cover,
+            ),
+            const SizedBox(height: GapSize.xl),
+            Text(
+              title,
+              style: Theme.of(context).textTheme.titleLarge,
+              textAlign: TextAlign.center,
+            ),
+            const SizedBox(height: GapSize.m),
```
Comment (oleksandrbalan-etn) [inline lib/ui/widgets/empty_page.dart:37]: Nit: The gap is a bit large (24 in app vs 16 in figma)

**Review by oleksandrbalan-etn — APPROVED** [review]
Just few nits 👍

File: lib/ui/screens/boxes_screen.dart (hunk 1 above)
Comment (fabiawrc) [reply lib/ui/screens/boxes_screen.dart]: Fixed

File: lib/ui/widgets/empty_page.dart (hunk 2 above)
Comment (fabiawrc) [reply lib/ui/widgets/empty_page.dart]: Fixed, thanks for your watchful eyes

**Pull request merged**